airflow/lineage/backend/atlas/__init__.py

```diff
@@ -1,102 +0,0 @@
-# -*- coding: utf-8 -*-
-#
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-#
-
-# pylint:disable=missing-docstring
-
-from atlasclient.client import Atlas
-from atlasclient.exceptions import HttpError
-
-from airflow.configuration import conf
-from airflow.lineage import datasets
-from airflow.lineage.backend import LineageBackend
-from airflow.lineage.backend.atlas.typedefs import operator_typedef
-from airflow.utils.timezone import convert_to_utc
-
-SERIALIZED_DATE_FORMAT_STR = "%Y-%m-%dT%H:%M:%S.%fZ"
-
-_username = conf.get("atlas", "username")
-_password = conf.get("atlas", "password")
-_port = conf.get("atlas", "port")
-_host = conf.get("atlas", "host")
-
-
-class AtlasBackend(LineageBackend):  # pylint:disable=missing-docstring
```

Comment:
where is the atlas backend now? Or is the lineage now just stored inside airflow metadata?

Reply:
The backend(s) will return but I want it to work cleanly in Airflow first. In any case I will expose the lineage information through a rest endpoint which you can use e.g. Admundsen to pick up the lineage by pulling. 

The backends will work two fold: 1) do push based lineage 2) allow for enrichment of the information available in aiflow. Think of In(Table("mytable", external=True)) that then resolves columns autmatically or other kind of information.